### PR TITLE
Add pull request guides

### PR DESCRIPTION
**Pull request by ErikSchierboom**
This PR adds to new documents:

- The Contributors' Guide to Pull Requests
- The Maintainers' Guide to Pull Requests

### CODE REVIEW

File: building/README.md [hunk 1]
```diff
@@ -6,17 +6,6 @@ It contains information and specifications for all various areas of Exercism as
 To get an idea where you could help, have a look at our [task list][tasks].
 
 Contributions to the code base are made via [GitHub][github].
-In case you are not familiar with GitHub and the contribution workflow, here are some resources that might help you getting started:
-
-- [Introduction to GitHub - Interactive Course][github-course]
-- [How To Create a Pull Request on GitHub - Tutorial][pr-tutorial]
-- [How To Make a Simple GitHub PR - Video by anthonywritescode][anthonywritescode]
-
-You can read more about how to make sure your contribution is welcome in the documentation for [How to make a great Pull Request][great-pr].
```
Comment (ErikSchierboom) [inline building/README.md:15]: I've moved this section tot `building/github/README.md`, effectively creating a separate `github` sub-section. This PR adds two documents to that new section: a contributors guide to PRs and a maintainers guide to PRs.

**Comment by BethanyG** [comment]
I don't know if we want to call them out specifically -- I assume they are referenced in the GitHub tutorials -- but the [GitHub Desktop](https://desktop.github.com/) and [GitHub CLI](https://cli.github.com/) tools are really helpful to contributors who are newer to `git` and newer to the Git Hub Forking workflow.

**Comment by ErikSchierboom** [comment]
@BethanyG I really like that idea! I've updated the PR.

File: building/github/README.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
+# Contributing via GitHub
+
+Contributions to the Exercism code base are made via [GitHub][github].
+
+As indicated by its name, GitHub uses [git][git] as its version control system.
+If you're unfamiliar with Git, the best online resource is [Pro Git][pro-git].
```
Comment (iHiD) [inline building/github/README.md:6]:
```suggestion
If you're unfamiliar with Git, we recommend looking at is [Pro Git][pro-git].
```

Let's not make a judgement on what's best etc, as even if it's right, it'll go stale.

**Review by iHiD — APPROVED** [review]
Looks solid! Thanks :)

**Pull request merged**